Sources/InfomaniakCore/Asynchronous/FlowToAsyncResult.swift

```diff
@@ -111,8 +110,10 @@ public final class FlowToAsyncResult<Success> {
 
     // MARK: Init
 
-    public init() {
-        // META keep SonarCloud happy
+    /// Init method
+    /// - Parameter qos: Optionally set a custom QoS used by underlying queues
+    public init(qos: DispatchQoS = .default) {
```

Comment:
We can specify QoS on `FlowToAsyncResult` to be consistent with other core types.